spec/API_specification/signatures/constants.py

```diff
@@ -0,0 +1,25 @@
+e = None
+"""
+IEEE 754 floating-point representation of Euler's constant.
+
+``e = 2.71828182845904523536028747135266249775724709369995...``
+"""
+
+inf = None
```

Comment:
Let's actually add the constants here as floats (like `inf = float('inf')`, `pi = 3.141592653589793`). Otherwise these show `inf = None` in the documentation, which is confusing. Since these are actually supposed to be Python floats, not array objects, there's no issue here with doing this.